tethys_services/utilities.py

```diff
@@ -1,6 +1,6 @@
 """
 ********************************************************************************
-* Name: utilities.py
```

Comment:
Revert this change The file is still called utilities.py

Reply:
? same as others above